packages/api/src/controllers/access-control.ts

```diff
@@ -150,6 +151,16 @@ app.post(
         res.status(204);
         return res.end();
       case "jwt":
+        if (req.body.jwt) {
+          const [res] = await db.accessControlKey.find({
```

Comment:
Another thing... This logic here basically means that we can't cache this API call in catalyst API. It would need to be a unique cache key for each and every user that plays the stream, which sounds like a lot.

Could we implement this blacklist on catalyst API side instead? It could just get a list of jwts as a config and we don't add any more IO in this flow. It's probably fine for fishtank if we update that config ourselves for now.